src/biome/text/api_new/pipeline.py

```diff
@@ -497,6 +497,23 @@ def type_name(self) -> str:
         """The pipeline name. Equivalent to task head name"""
         return self.head.__class__.__name__
 
+      
+    def _filter_parameters_by_predicate(
+        self, predicate: Callable[[], bool]
+    ) -> List[str]:
+        filtered_parameters = filter(predicate, self._model.named_parameters())
```

Comment:
@frascuchon For me `self._model.named_parameters()` returns a tuple, so `p.requires_grad` in the lambda functions fails ... i tested it with pytorch 1.2 and 1.4

Reply:
Yes, it was a refactor error. Fixed now